Ivan_Boichuk_1_HW.cpp

```diff
@@ -0,0 +1,77 @@
+
+
+#include <cmath>
+#include <iostream>
+
+int main() {
+
+  constexpr int errorExitCode{-1};
+  constexpr int successExitCode{};
+
+  int lowRangeA{2};
+  int highRangeA{255};
+  int a{};
+
+  constexpr auto incorrectInputErrorText{
+      "You entered an incorrect number, try entering the number again"};
+
+  std::cout << "Please enter number in range [" << lowRangeA << ".."
+            << highRangeA << "] for a\n";
+  std::cin >> a;
+
+  if (a < lowRangeA or a > highRangeA) {
+    std::cout << incorrectInputErrorText;
+    return errorExitCode;
+  }
+
+  int lowRangeB{2};
+  int highRangeB{255};
+  int b{};
+
+  std::cout << "Please enter number in range [" << lowRangeB << ".."
+            << highRangeB << "] for b\n";
+  std::cin >> b;
+
+  if (b < lowRangeB or b > highRangeB) {
+    std::cout << incorrectInputErrorText;
+    return errorExitCode;
+  }
+
+  int lowRangeC{2};
+  int highRangeC{255};
+  int c{};
+
+  std::cout << "Please enter number in range [" << lowRangeB << ".."
+            << highRangeB << "] for c\n";
+  std::cin >> c;
+
+  if (c < lowRangeC or c > highRangeC) {
+    std::cout << incorrectInputErrorText;
+    return errorExitCode;
+  }
+
+  long long D{static_cast<long long>(b * b) -
```

Comment:
я рекомендую сделать переменную D как const, так как ниже по коду она явно не будет нигде меняться, это добавит лучшую читабельность кода.
```suggestion
  const long long D{static_cast<long long>(b * b) -
```